lib/cli.rb

```diff
@@ -12,14 +21,8 @@
   end
 end.parse!
 
-main_currency = :BYN
-table = Array.new
-i = 0
-CSV.foreach(options[:file], headers: true, header_converters: :symbol) do |row|
-  puts("#{row[:type]}: #{row[:name]}:  #{row[:currency]}\t - #{row[:amount]}  #{row[:currency]}")
-  table[i] = {type: row[:type], name: row[:name], currency: row[:currency], amount: row[:amount]}
-  i = i+1
-end
+conn = PG::Connection.open( dbname: 'monmon', user: 'postgres')
+table = convert_keys(conn.exec("SELECT * from accounts"))
```

Comment:
This will delete CSV functionality for good. That's not said to do in the task, and I would not do that.

There are two ways as I see to live with both functionalities (DB and CSV):
1. Add env var (`--adapter=DB|CSV`, default is `DB`)
2. Create another `cli.rb` file (`cli_db.rb` for example)

I prefer 1.